CHANGES.rst

```diff
@@ -230,6 +230,7 @@ Bug fixes
 - ``astropy.io.ascii``
 
 - ``astropy.io.fits``
+  - Removed raising of ``AssertionError`` that could occur after closing or deleting compressed image data. [#4690]
```

Comment:
Oh, sorry, this should be in the changelog for v1.1.3, the next bug fix release (also put a newline after the section heading).